content/en/_index.md

```diff
@@ -19,8 +19,9 @@ cascade:
 
 <div class="jumbotron px-5" style="background-color: #F8F9FB; margin-bottom: 4rem;">
        <h1>Welcome to Cobalt Documentation</h1>
-       <p class="my-3">Deep dive into our documentation to explore Cobalt—your all-in-one platform for pentesting your software.</p>
-       <a class="btn btn-outline-primary rounded" href="https://www.cobalt.io/" target="_blank">Explore Cobalt »</a>
+       <h3>Deep Dive into Cobalt</h3>
```

Comment:
I think we might not need this extra title line of Deep Dive into Cobalt. The description below already does the job.